tests/appsec/test_asm_standalone.py

```diff
@@ -198,11 +248,19 @@ def test_upstream_appsec_propagation__no_attack__is_propagated_as_is__being_0(se
                 ["Datadog-Client-Computed-Stats", trueish,] in data["request"]["headers"] for trueish in ["yes", "true"]
             )
 
+        downstream_headers = json.loads(self.r.text)
+        assert downstream_headers["X-Datadog-Origin"] == "rum"
+        assert downstream_headers["X-Datadog-Parent-Id"] != "34343434"
+        assert "_dd.p.other=1" not in downstream_headers["X-Datadog-Tags"]
+        assert "_dd.p.appsec=1" in downstream_headers["X-Datadog-Tags"]
+        assert downstream_headers["X-Datadog-Sampling-Priority"] == "2"
```

Comment:
This test case doesn't make sense to me because appsec only sets manual_keep, so  `"x-datadog-sampling-priority": "0"` +  `"x-datadog-tags": "_dd.p.appsec=1"` doesn't exist today, and the resulting assertion where the dowstream prio becomes 2 is undefined today.

So IIUC this test case correctly, we must remove it.

Reply:
This test case should not occur in real life as if we set "_dd.p.appsec=1" in the tags, we should also set sampling priority to 2. But if at some point in the propagation this value gets lowered, this scenario makes sure that if receiving "_dd.p.appsec=1" in tags we will reset priority to 2, and propagate that value.

Reply:
is it possible to have the sampling prio lowered from 2? 🤔 I was assuming not indeed but I guess you can always set force drop in a local service but not sure what happens downstream in that case.

Reply:
yes, a force drop gets propagated the same as any other sampling priority